### PR TITLE
Live version updated (Github Pages)

### PR DESCRIPTION
**Pull request by mauriciomonta**
I deployed my web with github pages

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 ## Live Demo
 
-[Live Demo Link](https://mauriciomonta.github.io/web_portfolio/)
+[Live Demo Link](https://github.com/mauriciomonta/portfolio)
```
Comment (rotimiazeez) [inline readme.md:16]: - [ ] Kindly make sure the link provided is the live link to your web page and not your GitHub👍.

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @mauriciomonta 👋, 
## Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: readme.md (hunk 1 above)
Comment (mauriciomonta) [reply readme.md]: Did it!

**Review by Hamzaoutdoors — APPROVED** [review]
Hi  @mauriciomonta, 

## Congratulations! 🎉 Project Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀

To Highlight:

- [x] The Pull Request has a proper title and description. ✔️
- [x] You have added a descriptive Readme file. Good Job. ✔️
- [x] There is no linter error is present. ✔️
- [x] Project has met all the technical aspects you needed for this project! ✔️ 

_Cheers and Happy coding and Keep rocking!_ 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**